setup.py

```diff
@@ -128,7 +128,6 @@
     'requests',
     'GitPython!=2.1.4, !=2.1.5, !=2.1.6',  # For difftool taking git refs
     'jupyter_server',
-    'jupyter_server_mathjax>=0.2.2',
```

Comment:
I suggested considering this because MathJax4 implementation in JupyterLab 4 now ships all static assets by itself, so there is no need for it. Indeed The import in the server, `from jupyter_server_mathjax.app import STATIC_ASSETS_PATH` and related code would need to be removed too.

Reply:
Ah ok then let's clean that up :+1:

Reply:
@jtpio @krassowski Thanks for your comments. I will clean up and push.